bin/GameIndex.dbf

```diff
@@ -34191,22 +34297,20 @@ Name   = Dead or Alive 2
 Region = NTSC-U
 Compat = 5
 [patches = 23AF6876]
-//dvd version
-	// change TLSNDDRV RPC to blocking, better would be to move the wait to the top of the function
+// Dvd version.
```

Comment:
Well the Dvd part could be changed to DVD, same goes for CD but then we are just nitpicking things so it's fine as it is right now.